release/src/main/groovy/TestScripts.groovy

```diff
@@ -95,16 +116,35 @@ class TestScripts {
      }
    }
 
-   // Check for expected results in stdout of the last command
+   // Check for expected results in stdout of the last command, return boolean
+   public boolean seeKeyWord(String expected){
+     if (!var.lastText.contains(expected)){
```

Comment:
This doesn't really do anything as written. Even if you want to inject a predicate, 

Even if it is important to do it with a method, just `return var.lastText.contains(expected)`